metrics/http.go

```diff
@@ -5,8 +5,17 @@ import (
 	"net/http"
 )
 
-func HTTPHandler(w http.ResponseWriter, r *http.Request) {
-	s := GetStats()
+func GoStatsHandler(w http.ResponseWriter, r *http.Request) {
+	s := GetGoStats()
+	w.Header().Set("Content-Type", "application/json")
+	if err := json.NewEncoder(w).Encode(s); err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+}
+
+func PlasmaStatsHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
__[golint]__
* [golint] exported function PlasmaStatsHandler should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->